src/formats/PDB.cpp

```diff
@@ -640,76 +688,77 @@ static std::string to_pdb_index(int64_t value, uint64_t width) {
 }
 
 struct ResidueInformation {
-    std::string atom_hetatm{ "HETATM" };
-    std::string resname { "XXX" };
-    std::string resid { "  -1" };
-    std::string chainid { "X" };
-    std::string inscode { " " };
-    std::string comp_type { "" };
+    std::string atom_hetatm = "HETATM";
+    std::string resname;
+    std::string resid;
+    std::string chainid;
+    std::string insertion_code;
+    std::string composition_type;
+    std::string segment;
```

Comment:
I'm guessing this is always set?

Reply:
By default this will be an empty string, keeping compatibility with previous code